src/Congo.WebApi.Tests/Controllers/CartControllerTests.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using Congo.WebApi.Controllers;
+using FluentAssertions;
+using MediatR;
+using Microsoft.AspNetCore.Mvc;
+using Moq;
+using Xunit;
+
+namespace Congo.WebApi.Tests
+{
+    public class CartControllerTests
+    {
+        private readonly Mock<IMediator> mediatorMoq;
```

Comment:
This should follow the same naming pattern as the rest of the project.
```suggestion
        private readonly Mock<IMediator> _mediatorMoq;
```

Reply:
Refactored all tests to use this pattern.